ompi/communicator/comm_cid.c

```diff
@@ -1569,7 +1569,8 @@ static int ompi_comm_ft_allreduce_intra_nb(int *inbuf, int *outbuf, int count,
 static int ompi_comm_ft_allreduce_inter_nb(int *inbuf, int *outbuf, int count,
                                            struct ompi_op_t *op, ompi_comm_cid_context_t *cid_context,
                                            ompi_request_t **req) {
-    return MPI_ERR_UNSUPPORTED_OPERATION;
+    //TODO: CID_INTER_FT needs an implementation, using the non-ft for now...
+    return ompi_comm_allreduce_inter_nb(inbuf, outbuf, count, op, cid_context, req);
```

Comment:
```suggestion
    int rc = ompi_comm_allreduce_inter_nb(inbuf, outbuf, count, op, cid_context, req);
    return (rc == MPI_ERR_REVOKED)? MPI_ERR_UNSUPPORTED_OPERATION: rc;
```
Should help us understand that this fails after a fault because it is not implemented.

Reply:
I applied your suggestion to the next function  `ompi_comm_ft_allreduce_intra_pmix_nb()` as well. If that is not correct, let me know and I'll revert that change.